docroot/themes/contrib/civictheme/civictheme_library/components/00-base/menu/menu.twig

```diff
@@ -28,7 +28,11 @@
     {% if menu_level == 0 %}
       <ul class="civictheme-menu civictheme-menu--level-{{ menu_level }} {{ modifier_class }} " data-component-name="civictheme-menu" {{ attributes|default('') }}>
     {% else %}
-      <div class="civictheme-menu__sub-menu__wrapper civictheme-menu__sub-menu__wrapper--level-{{ menu_level }}">
+      {% if is_collapsible %}
```

Comment:
Only change to menu.twig is adding collapsible panel to sub-menu

Reply:
@richardgaunt 

menu structure generator should not have any knowledge of the presentation - it is just a menu structure generator

This has to be worked on more to remove modifications related to presentaion